qos-service-impl/src/main/java/com/palantir/atlasdb/qos/config/ThrottlingStrategy.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.qos.config;
+
+import java.util.List;
+import java.util.function.Supplier;
+
+public interface ThrottlingStrategy {
+    double getClientLimitMultiplier(Supplier<List<CassandraHealthMetricMeasurement>> metricMeasurements);
```

Comment:
arg could be generified to `Supplier<Collection<? extends CassandraHealthMetricMeasurement>> metricMeasurements` or `Supplier<Stream<? extends CassandraHealthMetricMeasurement>> metricMeasurements`